armi/physics/neutronics/fissionProductModel/lumpedFissionProduct.py

```diff
@@ -344,133 +298,17 @@ def getMassFrac(self, oldMassFrac=None):
 
         return massFrac
 
-    def setGasRemovedFrac(self, removedFrac):
-        """
-        Set the fraction of total fission gas that is removed from all LFPs.
-        """
-        for lfp in self.values():
-            lfp.setGasRemovedFrac(removedFrac)
-
-    def getGasRemovedFrac(self):
-        """
-        Get the fraction of total fission gas that is removed from all LFPs.
-        """
-        lastVal = -1
-        for lfp in self.values():
-            myVal = lfp.getGasRemovedFrac()
-            if lastVal not in (-1, lastVal):
-                raise RuntimeError(
-                    "Fission gas release fracs in {0} are decoupled" "".format(self)
-                )
-            lastVal = myVal
-        return lastVal
-
-
-class SingleLumpedFissionProductCollection(LumpedFissionProductCollection):
-    """
-    This is a subclass of LumpedFissionProductCollection to be used when you
-    want to collapse all the fission products into a single lumped fission product
-
-    There were numerous checks in places to ensure that a collection of
-    fission products only had 1 lfp and this object consolidates them.
-
-    Notes
-    -----
-    armi.physics.neutronics.fissionProductModel.lumpedFissionProduct.FissionProductDefinitionFile.createSingleLFPFromFile
-        is a factory for these
-
-    """
-
-    def __init__(self):
-        super(SingleLumpedFissionProductCollection, self).__init__()
-        self.collapsible = True
-
-    def getFirstLfp(self):
-        return list(self.values())[0]
-
-    def getName(self):
```

Comment:
This was stale/dead code that was implemented long ago and never used for anything practical.